lighthouse-core/audits/byte-efficiency/uses-optimized-animated-images.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * @license Copyright 2018 Google Inc. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+/*
+ * @fileoverview Audit a page to ensure that videos are used instead of animated gifs
+ */
+'use strict';
+
+const Audit = require('../audit');
+const WebInspector = require('../../lib/web-inspector');
+
+// the threshold for the size of GIFs wich we flag as unoptimized
+const GIF_BYTE_THRESHOLD = 10 * 1000;
+
+class UsesOptimizedAnimatedImages extends Audit {
+  /**
+   * @return {!AuditMeta}
+   */
+  static get meta() {
+    return {
+      name: 'uses-optimized-animated-images',
+      informative: true,
+      description: 'Use a video format for animated content',
+      helpText: 'no help here, your on your own!',
```

Comment:
my take: "Large GIFs are inefficient for delivering animated content. Consider using MPEG4/WebM videos for animations and PNG/WebP for static images instead of GIF to save network bytes. [Learn more](https://en.wikipedia.org/wiki/Video_alternative_to_GIF)."

wikipedia link